### DISCUSSION TITLE
Fixing error on eclipse

### DISCUSSION THREAD

**Discussion by saichandu06** [eclipse/windowbuilder#716] [General]
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/161620110/b8eee950-3936-43c9-b7a2-317a375d4a1d)
How to fix this error in eclipse

**Comment by ptziegler** [comment]
That looks a lot like a compilation error. What's the content of the problems view?

**Reply by saichandu06** [reply]
i want to create a new project by using windowBuilder,JFrame.problem is with drag and drop .

**Reply by ptziegler** [reply]
I need a little bit more information before I can tell what's going on:

- What Java version are you using?
- What WindowBuilder version are you using?
- Can you perhaps even zip and upload the project?

**Reply by saichandu06** [reply]
Java version - 18.0.2.1
How can i check windowBilder version

**Reply by ptziegler** [reply]
> How can i check windowBilder version

The version is normally shown in the bottom-right corner of the design editor:
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/6a1fc606-6acc-4b88-905b-d0fe6a8cb8f3)

Alternatively, you can also click on the "Help" item in the menu bar and then go About Eclipse IDE -> Installation Details -> Plug-ins
If you search for "Eclipse WindowBuilder", it should show you the version of the WindowBuilder bundles:
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/82af922e-6f4d-465f-8441-0c555eddc6c6)

However, I believe there is something wrong with the Eclipse project you're using. The easiest way to help you would be if you could zip and upload the project here.

**Comment by saichandu06** [comment]
![Uploading image.png…]()
WindowBilder version - 1.15.0.202402200547
and how can i upload my file ..

**Reply by ptziegler** [reply]
> Uploading image.png...

You need to wait a few seconds to make sure the image is actually uploaded.

>  and how can i upload my file ..

You can just drag & drop them in the comment box

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/9a4fb62b-fbff-4971-bf22-d1d7b0416c9d)

**Reply by saichandu06** [reply]
![Capture](https://github.com/eclipse-windowbuilder/windowbuilder/assets/161620110/d169c83e-ab5d-4eaa-a070-1111d3fdb415)

**Comment by saichandu06** [comment]
[RegistrationForm.zip](https://github.com/eclipse-windowbuilder/windowbuilder/files/14499420/RegistrationForm.zip)

**Reply by ptziegler** [reply]
Thanks! I had a look at it, but the problem seems to be fine as well. So I'm not really sure what else is causing the problem... 😦

Out of curiosity, the exception should be logged in the error log (which you can open via Window -> Show View -> Error Log). If you right-click on it, you can copy the error plus the complete stack trace. Can you perhaps post it here?
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/1fb3e38a-50d5-42c7-9e7b-aaa6904f6422)

Alternatively, does the issue also happen with a clean Eclipse installation?
i.e. if you download and start the vanilla SDK:
https://download.eclipse.org/eclipse/downloads/drops4/R-4.30-202312010110/download.php?dropFile=eclipse-SDK-4.30-win32-x86_64.zip

and then simply install the matching WindowBuilder release via `Help -> Install new software`:
https://download.eclipse.org/windowbuilder/updates/release/1.14.0
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/1da7d6f6-679f-470d-ba9d-52855c65578e)

Does the problem still show up?
